sonic-net/sonic-buildimage
# [Mellanox] Update Nvidia sai.profile SKU files to have common file

Depends on https://github.com/sonic-net/sonic-sairedis/pull/1352

Add new common sai.profile,
Update Nvidia/Mellanox sai.profile of all SKUs to remove common parameters.



#### Why I did it
To have the ability to add common parameters to only 1 file instead of all SKUs.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add a new common file with all common key-value parameters. 
also, updated syncd_init_common.sh to take common parameters from the new file in https://github.com/sonic-net/sonic-sairedis/pull/1352

#### How to verify it
Run the script with changed .profile files, and make sure all common parameters exist in final sai.profile. 


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)



- [x] master-internal-build

#### Description for the changelog

Add new common sai.profile,
Update Nvidia/Mellanox sai.profile of all SKUs to remove common parameters.


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
